rizzino_a05a.js

```diff
@@ -0,0 +1,21 @@
+/* Julie Rizzino
+rizzino_a05a.js
+INFO 2124
+Thoendel
+1/14/20 */
+
```

Comment:
no line by line comments! Bummer! As per the assignment - this will cost you 10%:
-2